service/scaladsl/api/src/main/scala/com/lightbend/lagom/scaladsl/api/deser/PathParamSerializer.scala

```diff
@@ -119,7 +121,7 @@ trait LowPriorityPathParamSerializers {
    */
   implicit def traversablePathParamSerializer[CC[X] <: Traversable[X], Param: PathParamSerializer](
       implicit delegate: PathParamSerializer[Param],
-      bf: CanBuildFrom[CC[_], Param, CC[Param]]
+      bf: Factory[Param, CC[Param]]
```

Comment:
`Factory` is only available in `2.13`. We'll probably need to split this file and have two versions: on for `2.11` and `2.12` and the other for `2.13`

Reply:
Rather than split the whole file just have a the definition of the `Factory` type available for 2.12 and import it.

Reply:
* https://github.com/playframework/play-json/blob/2.8.0-M1/play-json/shared/src/main/scala-2.13-/play/api/libs/json/ScalaCollectionCompat.scala
* https://github.com/playframework/play-json/blob/2.8.0-M1/play-json/shared/src/main/scala-2.13+/play/api/libs/json/ScalaCollectionCompat.scala
* `import ScalaCollectionCompat._`